src/main/jbake/content/blog/2017/03/15/hawkular-opentracing-otsc.adoc

```diff
@@ -0,0 +1,43 @@
+= Distributed Tracing Workshop and OpenTracing Collaboration
+Pavol Loffay
+2017-03-15
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+Distributed tracing workshop is organized by https://groups.google.com/forum/#!forum/distributed-tracing[Distributed Tracing Workgroup]
+and takes place approximately once a year. This time it was 18th-19th February in Berlin. It was a pleasure
+to be there with people from various companies and share ideas and a common goal to improve distributed tracing.
+
+== Summary
+
+We have discussed many thing here are my main takeaways:
+
+Key upcoming changes in OpenTracing:
+
+* Formal project organization OTSC (keep reading).
+* Clarification between logging, metrics and tracing. The discussion continues in https://github.com/opentracing/specification/pull/46[specification/pull/46].
+* In-process context propagation. This feature will be probably the biggest change in the next OpenTarcing
+ version. It should solve inconvenient passing of parent context. Users would just start new span and if there
+ is parent available it will be automatically linked.
+
+Trace context propagation:
+
+* Currently there are no standard headers for passing trace context. Several tracing systems adopted
+ OpenZipkin https://github.com/openzipkin/b3-propagation[B3 Propagation].
+ Standardization of context header should help with concerns like CORS and linking together traces between different
+ service providers. Proposed https://docs.google.com/document/d/1Mrw7hxVAkj7h98hvgRixDt1RrEJ5fsqqddPofIHRVvI/edit[document].
+
+== Hawkular on OpenTracing Specification Council!
+
+I am also happy to announce that Hawkular project is part of
+https://github.com/opentracing/specification/blob/master/project_organization.md[OpenTracing Specification Council] (OTSC).
+This more formal project organization structure will help with project growth and ensure that all major constituencies
+are involved.
```

Comment:
Can you drop some more words on how this is going to happen, also I understand this is brand new, explain a bit more what the council it? Of course you link to it, but giving a summary of this may nevertheless be a good idea.